Packs/XSOAR_Engineer_Training/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### XSOAR Engineer Training
+
+- Added the *body* argument to the *xet-send-mail*
```

Comment:
Do you mean to add the body argument without any changes to the .py file?
I see in the code that the given arguments have no meaning (The xet-send-mail command only prints a hardcoded string). Is that your intention?

Reply:
Yeah, the command doesn't really do anything, we use it for training customers as it simulates sending an email, without actually sending an email.  Updated the release notes file.